### ISSUE TITLE
empty jail creation gives traceback error

### ISSUE THREAD

**Issue by Zenzou** [iocage/iocage#512]
I'm currently on Freenas 11.1-U4 and trying out iocage.
I've successfully created jails using --plugins (deluge,emby), and created normal and basejails (unifi5 controller)
I tried to create a debian jail,
`http://iocage.readthedocs.io/en/latest/debian.html`
however 'iocage create -e' gives a traceback.
```
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/create.py", line 103, in cli
    elif release.lower() == "latest":
AttributeError: 'NoneType' object has no attribute 'lower'
```
Looks like they're using iocage 1.0 ALPHA

**Comment by jsegaert** [comment]
This particular traceback was already reported and fixed in #507.  
However, creating an empty jail using the latest version of iocage also seems to be an issue, but it produces a different traceback:
```
/root root@trinity#  iocage create -e -n debjail 
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/create.py", line 145, in cli
    basejail=basejail, empty=empty)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/iocage.py", line 644, in create
    exit_on_error=self.exit_on_error).create_jail()
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_create.py", line 86, in create_jail
    return self._create_jail(jail_uuid, location)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_create.py", line 348, in _create_jail
    with open(f"{location}/root/etc/hosts", "r") as _etc_hosts:
FileNotFoundError: [Errno 2] No such file or directory: '/iocage/jails/debjail/root/etc/hosts'
```

**Comment by gronke** [comment]
Appears to be fixed in #531 - please re-open if the issue persists. @jsegaert 💜for the PR!
